barretenberg/cpp/src/barretenberg/goblin/goblin.hpp

```diff
@@ -21,7 +21,7 @@
 
 namespace bb {
 
-class GoblinProver {
+class Goblin {
```

Comment:
Renamed this to just Goblin and got rid of GoblinVerifier since its logic is now just a single static `verify` method